web/public/mgmt/config.yml

```diff
@@ -1013,6 +1013,26 @@ collections:
                       name: "ariaContextWillLooseCalendarAndCertificationAccess",
                       widget: "string",
                     }
+                  - {
+                      label: "Aria Context - Stepper Labels",
+                      name: "ariaContextStepperLabels",
+                      widget: "string",
+                    }
+                  - {
+                      label: "Aria Context - Stepper State - Complete",
+                      name: "ariaContextStepperStateComplete",
+                      widget: "string",
+                    }
+                  - {
+                      label: "Aria Context - Stepper State - Error",
+                      name: "ariaContextStepperStateError",
+                      widget: "string",
+                    }
+                  - {
+                      label: "Aria Context - Stepper State - Incomplete",
+                      name: "ariaContextStepperStateIncomplete",
+                      widget: "string",
+                    }
```

Comment:
Is there a reason we'd want to expose this via the CMS? Should these just be hidden items?

Asking because if this is "standard" language and we don't anticipate a change, does it really matter?

Reply:
I'm not entirely sure how standard it is and my concern was it was something that might be changed in the future by content so I thought it might be good to expose.